### PR TITLE
Try ipyleaflet

### PR DESCRIPTION
**Pull request by martinRenou**
cc. @giswqs

### CODE REVIEW

File: examples/ipyleaflet.ipynb [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+    "from ipyleaflet import Map, GeoJSON, WidgetControl"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "data = pd.read_json('https://raw.githubusercontent.com/jupyter-widgets/ipyleaflet/master/examples/nations.json')"
```
Comment (jtpio) [inline examples/ipyleaflet.ipynb:41]: We probably still need to use the `from js import fetch` trick to fetch extra data for now:

https://github.com/jtpio/jupyterlite/issues/119#issuecomment-854495013

**Comment by jtpio** [comment]
Nice, thanks!

Looks like the latest `ipyleaflet` removes the hard dependency on `shapely`: https://github.com/jupyter-widgets/ipyleaflet/blob/master/CHANGELOG.md#v0140

And now shows up nicely on the RTD preview :tada: 

image

**Comment by jtpio** [comment]
Maybe we can move this `console.log`: 

https://github.com/jtpio/jupyterlite/blob/5f38f5cfbf373ad2c8a6c4bfd4f2d54124c36525/packages/pyolite-kernel/src/worker.ts#L270

Down to the execute request statement:

https://github.com/jtpio/jupyterlite/blob/5f38f5cfbf373ad2c8a6c4bfd4f2d54124c36525/packages/pyolite-kernel/src/worker.ts#L276

Since it seems to be flooding the dev tools with comm messages (for example when moving the mouse):

image

**Comment by martinRenou** [comment]
Cool! 

Now might be a good time to reorder notebooks in the examples directory

**Comment by giswqs** [comment]
How to access the JupyterLite RTD preview?

**Comment by jtpio** [comment]
@giswqs there is a link below in the checks:

image

**Comment by jtpio** [comment]
@martinRenou is this ready for review and merge?

Looks like it's working well on the RTD preview, and it would be a great example to add to the demo site.

**Comment by martinRenou** [comment]
Yep! Should we change the directory layout of the examples though? Would having a "Interactive Widgets" directory be a good idea?

Also maybe moving all pyolite examples in a separate dir?

**Comment by jtpio** [comment]
Sounds good.

It would be fine to do the example restructuring either here or in a separate PR.

**Comment by martinRenou** [comment]
Ok :) Let's do it in another PR then. And get the ipycanvas example in as well before the restructuring?

**Comment by jtpio** [comment]
> And get the ipycanvas example in as well before the restructuring?

Sounds good :+1:

**Review by jtpio — APPROVED** [review]
Thanks!

**Comment by martinRenou** [comment]
Thank you! :)

**Pull request merged**